### PR TITLE
Add required uri parameter block

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: ApiBundle/Transformer/BlockTransformer.php [hunk 1]
```diff
@@ -31,21 +32,24 @@ class BlockTransformer extends AbstractTransformer
...
+        BlockRepositoryInterface $blockRepository,
+        GenerateFormManager $generateFormManager
     ) {
         parent::__construct($facadeClass);
         $this->displayBlockManager = $displayBlockManager;
         $this->blockConfigurationManager = $blockConfigurationManager;
         $this->translator = $translator;
         $this->nodeRepository = $nodeRepository;
         $this->blockRepository = $blockRepository;
+        $this->generateFormManager = $generateFormManager;
```
Comment (alavieille) [inline ApiBundle/Transformer/BlockTransformer.php]: add property in class

File: BackofficeBundle/Resources/public/template/Node/nodeMessageToolbarView._tpl.html [hunk 2]
```diff
@@ -8,9 +8,16 @@
 <% }; %>
 <% if (false === node.get('rights').can_publish_node) {%>
 <div class="alert alert-warning" role="warning">
-    <div class="heading"><%- Translator.trans('open_orchestra_backoffice.node.route_pattern_arguments.title') %></div>
-    <p>
-        <%= (Translator.trans('open_orchestra_backoffice.node.route_pattern_arguments.message').replace(/\\n/g, '<br>')) %>
-    </p>
+    <% if (1 == requiredUriParameters.length) { %>
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/template/Node/nodeMessageToolbarView._tpl.html]: use pluralization

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**